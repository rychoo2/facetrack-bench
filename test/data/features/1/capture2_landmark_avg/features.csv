filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_eye_distance_x,rel_eye_distance_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191215123648.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9796834309895833,0.155859375,20191215123648000,0.368359375,-0.01597222222222222,0.23046875,0.43472222222222223,0.5186440677966102,0.7156549520766773,0.5661016949152542,-0.03035143769968051,0.45714285714285713,0.5303030303030303,,
images/20191215123629.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.0036878797743055557,0.01906105324074074,20191215123629000,,,,,,,,,,,,
images/20191215123639.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.14441460503472223,0.9633391203703704,20191215123639000,0.425,0.05694444444444444,0.2515625,0.44722222222222224,0.562111801242236,0.6770186335403726,0.531055900621118,-0.06832298136645963,0.5423728813559322,0.3888888888888889,0.4714285714285714,0.5333333333333333
images/20191215123634.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010639105902777777,0.6308424961419753,20191215123634000,0.411328125,0.06458333333333334,0.25625,0.45555555555555555,0.586890243902439,0.6310975609756098,0.5426829268292683,-0.053353658536585365,0.5748031496062992,0.4339622641509434,0.5138888888888888,0.5306122448979592
images/20191215123635.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.009682888454861111,0.755784625771605,20191215123635000,0.428125,0.03888888888888889,0.25703125,0.45694444444444443,0.5349544072948328,0.6413373860182371,0.5045592705167173,-0.06382978723404255,,,0.4566929133858268,0.48484848484848486
images/20191215123637.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007859971788194444,0.9523268711419753,20191215123637000,0.41875,0.03194444444444444,0.27421875,0.4875,0.5584045584045584,0.6410256410256411,0.4757834757834758,-0.06267806267806268,0.5299145299145299,0.3888888888888889,,
images/20191215123636.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007587348090277778,0.9118272569444444,20191215123636000,,,,,,,,,,,,
images/20191215123633.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.008323838975694444,0.4863474151234568,20191215123633000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.6366459627329193,0.6987577639751553,0.45962732919254656,-0.031055900621118012,0.6373626373626373,0.1794871794871795,0.4881889763779528,0.4358974358974359
images/20191215123631.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010767957899305555,0.29573688271604937,20191215123631000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.6459627329192547,0.6583850931677019,0.4782608695652174,-0.027950310559006212,0.6304347826086957,0.30434782608695654,0.5109489051094891,0.4782608695652174
images/20191215123643.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.7875189887152778,0.9573591820987655,20191215123643000,0.376171875,0.06527777777777778,0.256640625,0.45625,0.4855403348554033,0.6666666666666666,0.5494672754946728,-0.0441400304414003,0.46153846153846156,0.4722222222222222,0.42748091603053434,0.4444444444444444
images/20191215123642.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.6603339301215277,0.9516541280864198,20191215123642000,0.3859375,0.07083333333333333,0.251171875,0.4465277777777778,0.49611197511664074,0.6687402799377916,0.552099533437014,-0.038880248833592534,0.4825174825174825,0.475,0.45454545454545453,0.525
images/20191215123640.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.3508992513020833,0.9571566358024691,20191215123640000,0.4125,0.07847222222222222,0.24765625,0.44027777777777777,0.5220820189274448,0.6750788643533123,0.5425867507886435,-0.07097791798107256,0.5298507462686567,0.3888888888888889,0.45714285714285713,0.37037037037037035
images/20191215123641.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.47699924045138886,0.9544077932098766,20191215123641000,0.38984375,0.07916666666666666,0.241796875,0.42986111111111114,0.5121163166397416,0.6672051696284329,0.5718901453957996,-0.045234248788368334,0.5244755244755245,0.3888888888888889,,
images/20191215123645.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801947699652778,0.7978804976851852,20191215123645000,,,,,,,,,,,,
images/20191215123651.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.8140584309895833,0.010956790123456791,20191215123651000,0.387890625,0.029861111111111113,0.23125,0.4111111111111111,0.49493243243243246,0.6148648648648649,0.581081081081081,-0.037162162162162164,0.46258503401360546,0.5084745762711864,,
images/20191215123650.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9923570421006944,0.026126060956790122,20191215123650000,0.35546875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5372670807453416,0.729050279329609,0.5372670807453416,-0.04189944134078212,0.45,0.5362318840579711,0.3884297520661157,0.5072463768115942
images/20191215123644.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9668429904513889,0.9584104938271605,20191215123644000,0.383984375,0.06597222222222222,0.21953125,0.3909722222222222,0.4679715302491103,0.6856127886323268,0.5711743772241993,-0.04085257548845471,0.48905109489051096,0.3333333333333333,,
images/20191215123652.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,,,,0.391796875,0.03125,0.233984375,0.41597222222222224,0.5208681135225376,0.6193656093489148,0.6076794657762938,-0.04006677796327212,0.4857142857142857,0.5454545454545454,0.4444444444444444,0.5303030303030303
images/20191215123646.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.98377685546875,0.6164086612654321,20191215123646000,0.3875,0.04861111111111111,0.2046875,0.3638888888888889,0.5152671755725191,0.683206106870229,0.5648854961832062,-0.030534351145038167,0.5470085470085471,0.3055555555555556,0.3162393162393162,0.30303030303030304
images/20191215123647.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801201714409722,0.29488088348765434,20191215123647000,0.38359375,0.03680555555555556,0.232421875,0.4131944444444444,0.45546218487394957,0.6369747899159663,0.5546218487394958,-0.03529411764705882,0.44285714285714284,0.5178571428571429,0.3937007874015748,0.49056603773584906
