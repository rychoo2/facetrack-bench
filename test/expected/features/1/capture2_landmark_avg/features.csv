filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_left_eye_x,rel_left_eye_y,rel_right_eye_x,rel_right_eye_y,rel_left_pupil_face_x,rel_left_pupil_face_y,rel_eye_distance_x,rel_eye_distance_y,rel_right_pupil_face_x,rel_right_pupil_face_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191215123648.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9796834309895833,0.155859375,20191215123648000,0.36875,-0.008333333333333333,0.22734375,0.42916666666666664,0.520618556701031,0.7022653721682848,0.563573883161512,0.2815533980582524,0.140893470790378,0.32038834951456313,0.7285223367697594,0.3559870550161812,0.5618556701030928,-0.042071197411003236,0.281786941580756,0.3883495145631068,0.4444444444444444,0.5111111111111111,0.3942307692307692,0.4666666666666667
images/20191215123629.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.0036878797743055557,0.01906105324074074,20191215123629000,,,,,,,,,,,,,,,,,,,,
images/20191215123639.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.14441460503472223,0.9633391203703704,20191215123639000,0.4171875,0.07222222222222222,0.25234375,0.4486111111111111,0.5851393188854489,0.6594427244582043,0.5634674922600619,0.26625386996904027,0.1609907120743034,0.3281733746130031,0.7244582043343654,0.29102167182662536,0.5371517027863777,-0.06656346749226007,0.3219814241486068,0.35294117647058826,0.5591397849462365,0.32,0.47706422018348627,0.4
images/20191215123634.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010639105902777777,0.6308424961419753,20191215123634000,0.413671875,0.06527777777777778,0.254296875,0.45208333333333334,0.5637480798771122,0.6221198156682027,0.5453149001536098,0.22427035330261136,0.130568356374808,0.2764976958525346,0.7296466973886329,0.27035330261136714,0.5514592933947773,-0.059907834101382486,0.3241167434715822,0.32872503840245776,0.594059405940594,0.4166666666666667,0.525,0.53125
images/20191215123635.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.009682888454861111,0.755784625771605,20191215123635000,0.430859375,0.06597222222222222,0.25546875,0.45416666666666666,0.5382262996941896,0.6024464831804281,0.5122324159021406,0.24617737003058104,0.11467889908256881,0.2798165137614679,0.6834862385321101,0.26146788990825687,0.5290519877675841,-0.05198776758409786,0.28287461773700306,0.3165137614678899,0.6363636363636364,0.2,0.5092592592592593,0.48
images/20191215123637.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007859971788194444,0.9523268711419753,20191215123637000,0.43203125,0.06875,0.2546875,0.4527777777777778,0.5598159509202454,0.6165644171779141,0.5276073619631901,0.24386503067484663,0.13190184049079753,0.2898773006134969,0.6901840490797546,0.26226993865030673,0.5322085889570553,-0.0598159509202454,0.29754601226993865,0.3236196319018405,0.5760869565217391,0.25,0.5,0.4583333333333333
images/20191215123636.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007587348090277778,0.9118272569444444,20191215123636000,0.42734375,0.06527777777777778,0.26015625,0.4625,0.5615615615615616,0.6246246246246246,0.5345345345345346,0.24624624624624625,0.14714714714714713,0.2972972972972973,0.6906906906906907,0.27627627627627627,0.5225225225225225,-0.052552552552552555,0.3063063063063063,0.3333333333333333,0.5652173913043478,0.35714285714285715,0.49074074074074076,0.48
images/20191215123633.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.008323838975694444,0.4863474151234568,20191215123633000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.6335403726708074,0.6956521739130435,0.5900621118012422,0.27639751552795033,0.19254658385093168,0.32608695652173914,0.7391304347826086,0.3136645962732919,0.4937888198757764,-0.055900621118012424,0.35714285714285715,0.37267080745341613,0.6233766233766234,0.3333333333333333,0.5047619047619047,0.46875
images/20191215123631.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010767957899305555,0.29573688271604937,20191215123631000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.639751552795031,0.6583850931677019,0.5838509316770186,0.2360248447204969,0.19875776397515527,0.2546583850931677,0.7391304347826086,0.2639751552795031,0.4813664596273292,-0.024844720496894408,0.36024844720496896,0.3136645962732919,0.6493506493506493,0.25,0.5,0.475
images/20191215123643.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.7875189887152778,0.9573591820987655,20191215123643000,0.38359375,0.0875,0.2515625,0.44722222222222224,0.468944099378882,0.6273291925465838,0.48757763975155277,0.2236024844720497,0.08074534161490683,0.2608695652173913,0.6428571428571429,0.2608695652173913,0.5838509316770186,-0.043478260869565216,0.2236024844720497,0.2950310559006211,0.4424778761061947,0.42857142857142855,0.42201834862385323,0.44
images/20191215123642.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.6603339301215277,0.9516541280864198,20191215123642000,0.384765625,0.07361111111111111,0.253125,0.45,0.49382716049382713,0.6635802469135802,0.5138888888888888,0.24074074074074073,0.09104938271604938,0.2808641975308642,0.6805555555555556,0.29012345679012347,0.5694444444444444,-0.046296296296296294,0.25771604938271603,0.3271604938271605,0.4778761061946903,0.5,0.46551724137931033,0.5172413793103449
images/20191215123640.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.3508992513020833,0.9571566358024691,20191215123640000,0.42109375,0.1076388888888889,0.2328125,0.4131944444444444,0.511744966442953,0.6386554621848739,0.5033557046979866,0.21008403361344538,0.10067114093959731,0.2773109243697479,0.6912751677852349,0.23697478991596638,0.5637583892617449,-0.07226890756302522,0.2348993288590604,0.29747899159663865,0.5773195876288659,0.27586206896551724,0.4,0.2857142857142857
images/20191215123641.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.47699924045138886,0.9544077932098766,20191215123641000,0.38828125,0.08125,0.243359375,0.4326388888888889,0.507223113964687,0.6452648475120385,0.5329052969502408,0.24558587479935795,0.10914927768860354,0.29052969502407705,0.7158908507223114,0.29052969502407705,0.5714285714285714,-0.051364365971107544,0.27608346709470305,0.3290529695024077,0.504424778761062,0.4827586206896552,0.46017699115044247,0.42857142857142855
images/20191215123645.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801947699652778,0.7978804976851852,20191215123645000,0.35546875,0.0375,0.2515625,0.44722222222222224,0.546583850931677,0.717391304347826,0.5683229813664596,0.2795031055900621,0.17080745341614906,0.32298136645962733,0.7080745341614907,0.3416149068322981,0.5683229813664596,-0.049689440993788817,0.39751552795031053,0.38509316770186336,0.41284403669724773,0.5555555555555556,0.6952380952380952,0.6060606060606061
images/20191215123651.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.8140584309895833,0.010956790123456791,20191215123651000,0.3875,0.029861111111111113,0.234375,0.4166666666666667,0.485,0.6083333333333333,0.5166666666666667,0.17166666666666666,0.09,0.205,0.68,0.24166666666666667,0.5666666666666667,-0.03833333333333333,0.23333333333333334,0.2783333333333333,0.44954128440366975,0.4772727272727273,0.3944954128440367,0.4888888888888889
images/20191215123650.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9923570421006944,0.026126060956790122,20191215123650000,0.36875,-0.022222222222222223,0.23046875,0.43472222222222223,0.5152542372881356,0.6964856230031949,0.5525423728813559,0.2747603833865815,0.13898305084745763,0.3194888178913738,0.711864406779661,0.35782747603833864,0.5627118644067797,-0.04313099041533546,0.2677966101694915,0.3929712460063898,0.44761904761904764,0.5306122448979592,0.38,0.4791666666666667
images/20191215123644.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9668429904513889,0.9584104938271605,20191215123644000,0.378515625,0.05138888888888889,0.2359375,0.4201388888888889,0.45364238410596025,0.6958677685950413,0.5016556291390728,0.32727272727272727,0.15397350993377484,0.3603305785123967,0.6705298013245033,0.34710743801652894,0.5562913907284768,-0.02975206611570248,0.3360927152317881,0.3933884297520661,0.504950495049505,0.24,0.625,0.4166666666666667
images/20191215123652.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,,,,0.3890625,0.027083333333333334,0.237109375,0.4215277777777778,0.5271828665568369,0.6210873146622735,0.5436573311367381,0.17627677100494235,0.11532125205930807,0.22240527182866557,0.7182866556836903,0.23887973640856672,0.5963756177924218,-0.04612850082372323,0.27347611202635913,0.2751235584843493,0.49074074074074076,0.475,0.44036697247706424,0.43243243243243246
images/20191215123646.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.98377685546875,0.6164086612654321,20191215123646000,0.3859375,0.059027777777777776,0.212890625,0.37916666666666665,0.48807339449541287,0.6868131868131868,0.5321100917431193,0.21794871794871795,0.09908256880733946,0.2619047619047619,0.708256880733945,0.29120879120879123,0.6128440366972477,-0.038461538461538464,0.22752293577981653,0.3241758241758242,0.45714285714285713,0.5,0.36082474226804123,0.4594594594594595
images/20191215123647.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801201714409722,0.29488088348765434,20191215123647000,0.387890625,0.06111111111111111,0.20390625,0.36319444444444443,0.49233716475095785,0.6577437858508605,0.5383141762452107,0.18738049713193117,0.08237547892720307,0.2294455066921606,0.7260536398467433,0.26003824091778205,0.6379310344827587,-0.04206500956022945,0.22796934865900384,0.2944550669216061,0.4537037037037037,0.475,0.3917525773195876,0.425
